src/webui/src/components/trial-detail/TableList.tsx

```diff
@@ -458,7 +502,14 @@ class TableList extends React.Component<TableListProps, TableListState> {
                         className="titleColumn"
                     />
                 </Modal>
+                {/* compare trials based message */}
                 <Compare compareRows={selectRows} visible={isShowCompareModal} cancelFunc={this.hideCompareModal} />
+                {/* clone trial parameters and could submit a customized trial */}
+                <Customize
+                    visible={isShowCustomizedModal}
+                    hyperParameter={trialParameter}
```

Comment:
I think it's good to pass as few props as possible. `Customize` only needs to know which trial (ID) it's copying.

Reply:
Ok.